src/main/webapp/user-settings.jsp

```diff
@@ -40,19 +41,19 @@
         <tr>
           <td>Top Skill:</td><br>
           <td>
-            <input type="radio" value="NONE" name="skills" checked>
+            <input type="radio" value="NONE" name="skills" <%=checkedStatus[0]%>>
             <label>None</label><br>
-            <input type="radio" value="LEADERSHIP" name="skills">
+            <input type="radio" value="LEADERSHIP" name="skills" <%=checkedStatus[1]%>>
```

Comment:
As per comment above, clarity can be improved if this was something like `<%=checkedStatus["LEADERSHIP"]%>`